From f4019d96a708b09a804f59667d5dfd2a9eaaced5 Mon Sep 17 00:00:00 2001
From: Chase Maupin <Chase.Maupin@ti.com>
Date: Mon, 27 Jun 2011 13:33:43 -0500
Subject: [PATCH 1/3] OCF support: remove support for non-TI hardware

* Base OCF patch was updated to not patch in support for non-TI
  hardware.
* Makefile and Kconfig needed updating to remove these options

Upstream-Status: Inappropriate [TI SDK specific]

Signed-off-by: Chase Maupin <Chase.Maupin@ti.com>
---
 crypto/ocf/Kconfig  |   77 ---------------------------------------------------
 crypto/ocf/Makefile |   11 -------
 2 files changed, 0 insertions(+), 88 deletions(-)

diff --git a/crypto/ocf/Kconfig b/crypto/ocf/Kconfig
index b9c24ff..805cb4c 100644
--- a/crypto/ocf/Kconfig
+++ b/crypto/ocf/Kconfig
@@ -31,83 +31,6 @@ config OCF_CRYPTOSOFT
 	  A software driver for the OCF framework that uses
 	  the kernel CryptoAPI.
 
-config OCF_SAFE
-	tristate "safenet (HW crypto engine)"
-	depends on OCF_OCF
-	help
-	  A driver for a number of the safenet Excel crypto accelerators.
-	  Currently tested and working on the 1141 and 1741.
-
-config OCF_IXP4XX
-	tristate "IXP4xx (HW crypto engine)"
-	depends on OCF_OCF
-	help
-	  XScale IXP4xx crypto accelerator driver.  Requires the
-	  Intel Access library.
-
-config OCF_IXP4XX_SHA1_MD5
-	bool "IXP4xx SHA1 and MD5 Hashing"
-	depends on OCF_IXP4XX
-	help
-	  Allows the IXP4xx crypto accelerator to perform SHA1 and MD5 hashing.
-	  Note: this is MUCH slower than using cryptosoft (software crypto engine).
-
-config OCF_HIFN
-	tristate "hifn (HW crypto engine)"
-	depends on OCF_OCF
-	help
-	  OCF driver for various HIFN based crypto accelerators.
-	  (7951, 7955, 7956, 7751, 7811)
-
-config OCF_HIFNHIPP
-	tristate "Hifn HIPP (HW packet crypto engine)"
-	depends on OCF_OCF
-	help
-	  OCF driver for various HIFN (HIPP) based crypto accelerators
-	  (7855)
-
-config OCF_TALITOS
-	tristate "talitos (HW crypto engine)"
-	depends on OCF_OCF
-	help
-	  OCF driver for Freescale's security engine (SEC/talitos).
-
-config OCF_PASEMI
-	tristate "pasemi (HW crypto engine)"
-	depends on OCF_OCF && PPC_PASEMI
-	help
-	  OCF driver for the PA Semi PWRficient DMA Engine
-
-config OCF_EP80579
-	tristate "ep80579 (HW crypto engine)"
-	depends on OCF_OCF
-	help
-	  OCF driver for the Intel EP80579 Integrated Processor Product Line.
-
-config OCF_CRYPTOCTEON
-	tristate "cryptocteon (HW crypto engine)"
-	depends on OCF_OCF
-	help
-	  OCF driver for the Cavium OCTEON Processors.
-
-config OCF_KIRKWOOD
-	tristate "kirkwood (HW crypto engine)"
-	depends on OCF_OCF
-	help
-	  OCF driver for the Marvell Kirkwood (88F6xxx) Processors.
-
-config OCF_C7108
-	tristate "Micronas 7108 (HW crypto engine)"
-	depends on OCF_OCF
-	help
-	  OCF driver for the Microna 7108 Cipher processors.
-
-config OCF_OCFNULL
-	tristate "ocfnull (fake crypto engine)"
-	depends on OCF_OCF
-	help
-	  OCF driver for measuring ipsec overheads (does no crypto)
-
 config OCF_BENCH
 	tristate "ocf-bench (HW crypto in-kernel benchmark)"
 	depends on OCF_OCF
diff --git a/crypto/ocf/Makefile b/crypto/ocf/Makefile
index fa951f4..aab3fae 100644
--- a/crypto/ocf/Makefile
+++ b/crypto/ocf/Makefile
@@ -37,17 +37,6 @@ obj-$(CONFIG_OCF_CRYPTODEV)   += cryptodev.o
 obj-$(CONFIG_OCF_CRYPTOSOFT)  += cryptosoft.o
 obj-$(CONFIG_OCF_BENCH)       += ocf-bench.o
 
-$(_obj)-$(CONFIG_OCF_SAFE)    += safe$(_slash)
-$(_obj)-$(CONFIG_OCF_HIFN)    += hifn$(_slash)
-$(_obj)-$(CONFIG_OCF_IXP4XX)  += ixp4xx$(_slash)
-$(_obj)-$(CONFIG_OCF_TALITOS) += talitos$(_slash)
-$(_obj)-$(CONFIG_OCF_PASEMI)  += pasemi$(_slash)
-$(_obj)-$(CONFIG_OCF_EP80579) += ep80579$(_slash)
-$(_obj)-$(CONFIG_OCF_CRYPTOCTEON) += cryptocteon$(_slash)
-$(_obj)-$(CONFIG_OCF_KIRKWOOD) += kirkwood$(_slash)
-$(_obj)-$(CONFIG_OCF_OCFNULL) += ocfnull$(_slash)
-$(_obj)-$(CONFIG_OCF_C7108) += c7108$(_slash)
-
 ocf-objs := $(OCF_OBJS)
 
 $(list-multi) dummy1: $(ocf-objs)
-- 
1.7.0.4

